kubernetes/omsagent.yaml

```diff
@@ -582,20 +582,20 @@ spec:
    name: omsagent-win
    namespace: kube-system
    labels:
-    component: oms-agent
-    tier: node
+    component: oms-agent-win
+    tier: node-win
   spec:
    updateStrategy:
     type: RollingUpdate
    selector:
     matchLabels:
-      component: oms-agent
-      tier: node
+      component: oms-agent-win
+      tier: node-win
    template:
     metadata:
       labels:
-        component: oms-agent
-        tier: node
+        component: oms-agent-win

```

Comment:
do you know if '-win' was missing before, or is it a merge issue ? Is it '-win' in RP ?

Reply:
yeah, -win is there in the RP. I think it got missed when it was added to Docker-Provider so I'm adding it back here.

The current test I've done is with the -win label being present.

Reply:
ok thanks. As long as we keep the labels the same between linux & windows in each yaml (RP,AKS-E,HELM), and not copy between them, we are good.